stdlib/2.7/argparse.pyi

```diff
@@ -125,7 +125,7 @@ class _ActionsContainer:
     def get_default(self, dest): ...
     def add_argument(self,
         *args: str,
-        action: str = ...,
+        action: Union[str,Action] = ...,
```

Comment:
You need a space after the comma here (in general that's PEP 8 policy). (Also in the other file.)